frontend/src/app/esri-map/esri-map.component.ts

```diff
@@ -12,17 +12,16 @@ export class EsriMapComponent implements AfterViewInit {
 
   @ViewChild('mapViewNode') private mapViewEl: ElementRef;
 
-  constructor(public router: Router) { }
+  constructor(
```

Comment:
Purely for my own education, why did you change the format here?

Reply:
I typically break the constructor lines if we are adding multiple items. In this case I was importing firewoodInfoService to .getAll() and loop thru the forests that way, but esri hated that so I had to remove it